### PR TITLE
add the menu

### PR DESCRIPTION
**Pull request by koval1off**

### CODE REVIEW

File: bank.py [hunk 1]
```diff
@@ -6,9 +6,36 @@
...
+    # else:
+    #     print(f"The new balance is {user_balance}")
+    menu()
+
+def show_balance():
+    print(f"You choosed the Show balance operation!\nYour current balance is {user_balance}")
+    menu()
+
+def menu():
+    print("Menu:\n\t1) Change the balance\n\t2) Show new balance")   # show the menu operations
```
Comment (Alex-CourseAndMaterials) [inline bank.py:29]: use f string here

File: bank.py [hunk 2]
```diff
@@ -6,9 +6,36 @@
...
+    balance_changes = input("Enter changes you wanna make: ")
+    user_balance += int(balance_changes)   # upgrade balance
+    if user_balance < 0:
+        print("Too much to withdraw!")
+    # else:
+    #     print(f"The new balance is {user_balance}")
+    menu()
+
+def show_balance():
+    print(f"You choosed the Show balance operation!\nYour current balance is {user_balance}")
```
Comment (Alex-Krykun) [inline bank.py]: method should do only one action as in name

File: bank.py [hunk 3]
```diff
@@ -6,9 +6,36 @@
...
+    menu()
+
+def menu():
+    print("Menu:\n\t1) Change the balance\n\t2) Show new balance")   # show the menu operations
+    menu_choise = input("Enter the number of operation: ")
+    if int(menu_choise) == 1:
+        change_balance()
+    elif int(menu_choise) == 2:
+        show_balance()
+
```
Comment (Alex-Krykun) [inline bank.py]: add else case

File: bank.py [hunk 4]
```diff
@@ -6,9 +6,36 @@
 title = "ATM"
 print(title.center(9,'*'))      # making title
 
+user_balance = 5000
 user_id = input("Enter your ID: ")
 user_pass = input("Enter you password: ")
+
+def change_balance():
+    print("You choosed change balance operation!")
+    global user_balance
```
Comment (Alex-CourseAndMaterials) [inline bank.py:15]: do you really need to use global here?

File: bank.py [hunk 5]
```diff
@@ -6,9 +6,36 @@
...
+    balance_changes = input("Enter changes you wanna make: ")
+    user_balance += int(balance_changes)   # upgrade balance
+    if user_balance < 0:
```
Comment (Alex-Krykun) [inline bank.py]: check operation before actually do it

**Pull request merged**